tests/qos/test_qos_sai.py

```diff
@@ -0,0 +1,461 @@
+"""
+    QoS SAI feature in SONiC.
+
+    Parameters:
+        --ptf_portmap <filename> (str): file name of port index to DUT interface alias map. Default is None.
```

Comment:
This is a mandatory parameter. This varies for different sku's so its better not to generate a default and error out in the beginning if this is not specified

Reply:
It is mapping of port indexes to aliases. Port indexes are used in the ptf to target specific dut port. I do not think it is required.